var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -235,9 +234,7 @@ class LlvmAmdgpu(CMakePackage):
         resource(
             name="hsa-runtime",
             placement="hsa-runtime",
-            url="https://github.com/RadeonOpenCompute/ROCR-Runtime/archive/rocm-{0}.tar.gz".format(
-                d_version
-            ),
+            url="https://github.com/RadeonOpenCompute/ROCR-Runtime/archive/rocm-{d_version}.tar.gz",
```

Comment:
Apologies if I forgot to include the `f` in my suggestions:

```suggestion
            url=f"https://github.com/RadeonOpenCompute/ROCR-Runtime/archive/rocm-{d_version}.tar.gz",
```